include/ignition/gui/qml/PlottingInterface.qml

```diff
@@ -0,0 +1,556 @@
+              }
+
+          }
+
+          Rectangle {
+              color: (Material.theme == Material.Light) ? Material.color(Material.Grey, Material.Shade200) :
+                                                          Material.color(Material.Grey, Material.Shade900)
+              anchors.fill: parent
+              id: exportWidget
+
+
+              // make it scrolable
+              ScrollView {
+                  width: parent.width
+                  height: parent.height - exportBtn.height * 1.5
+                  ScrollBar.vertical.policy: ScrollBar.AlwaysOn
+                  clip: true
+
+                  Grid {
+                      id: grid
+                      anchors.fill: parent
+                      spacing: 10
+                      padding: 30
+                      columns: (exportWidget.width)/ 300
+                  }
+              }
+
+              ComboBox {
+                  id: exportBtn
+                  displayText: "Export to"
+                  model: ["CSV"]
+
+                  background: Rectangle {
+                      implicitWidth: 120
+                      implicitHeight: 40
+                      radius: 10
+                      color: Material.color(Material.Cyan, Material.Shade400)
+                  }
+                  anchors.bottom: parent.bottom
+                  anchors.right: parent.right
+                  anchors.margins: 10
+
+                  onActivated: {
+                      fileDialog.open();
+                  }
+              }
+              Rectangle {
+                  id: cancelBtn
+                  color: Material.color(Material.Grey, Material.Shade600);
+                  implicitWidth: 120
+                  implicitHeight: 40
+                  radius: 10
+                  anchors.bottom: parent.bottom
+                  anchors.left: parent.left
+                  anchors.margins: 10
+                  Text {
+                      text: "Cancel"
+                      anchors.centerIn: parent
+                      color: "white"
+                  }
+                  MouseArea {
+                      anchors.fill: parent
+                      hoverEnabled: true
+                      onEntered: parent.opacity = 0.9; cursorShape: Qt.PointingHandCursor
+                      onExited: parent.opacity = 1;
+                      onClicked: exportApp.close()
+                  }
+              }
+          }
+
+          FolderDialog {
+              id: fileDialog
+              title: "Choose a folder"
+              visible: false
+              options: FolderDialog.ShowDirsOnly
+
+              onAccepted: {
+                  if (exportBtn.currentText == "CSV")
+                  {
+                      var success = exportApp.exportCSV(folder);
+                      if (success)
+                          exportApp.close();
+                  }
+                  else if (exportBtn.currentText == "PDF")
```

Comment:
Let's remove all mentions of PDF because it's not supported yet. They should all be added at the same time once the feature is added.